src/main/java/com/google/cloud/teleport/splunk/SplunkEventWriter.java

```diff
@@ -106,6 +106,7 @@ public static Builder newBuilder() {
   public void setup() {
 
     checkArgument(url().isAccessible(), "url is required for writing events.");
+    checkArgument(url().get().startsWith("http"), "url has no protocol.");
```

Comment:
This check is not required since the format is being validated while building instance

Reply:
The other checks won't be evaluated if the pipeline is run from a template